Content/minimal/.gitignore

```diff
@@ -13,5 +13,6 @@ node_modules/
 release.cmd
 release.sh
 *.orig
+*.fsproj.user
 *.DotSettings.user
-*.fs.js
\ No newline at end of file
+*.fs.js
```

Comment:
We probably want `fs.js` in both minimal and full.

Reply:
agree, not sure why this was removed actually